src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM_DISCOUNT = 50;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
```

Comment:
Immutable constants are best defined outside the function definition

Reply:
@roman-mirzoian not in this case, because if you export a function it won't work anywhere else